next.config.mjs

```diff
@@ -44,7 +44,7 @@ const nextConfig = {
           loader: '@svgr/webpack',
           options: {
             prettier: false,
-            svgo: true,
+            svgo: false,
```

Comment:
Had to disable svgo because it minifies clip-path ids so that there is only one clip-path allowed on a given page. Thats why no icon was visible on the address book page so far.